README.md

```diff
@@ -572,6 +572,8 @@ You could also set the severity of `info-contact` explicitly to `error`, `warn`,
 
 #### Custom Spectral rules
 
+The Validator implements some validations as custom Spectral rules, [documented here](docs/spectral-rules.md).
```

Comment:
I think this is in the wrong place.  From the user's point of view, the rules we are adding are "standard rules" of our "ibm:oas" ruleset.  So this belongs in the previous section.